integrations/azure/README.md

```diff
@@ -0,0 +1,47 @@
+# Azure
```

Comment:
More documentation please about how do i know which resources are supported and how to map